### ISSUE TITLE
How to change background in dir tree of filedialog  and remove borders of dropdown listdir?

### ISSUE THREAD

**Issue by foozzi** [rdbende/Azure-ttk-theme#53]
1. ![Screenshot_20221130_163614](https://user-images.githubusercontent.com/1178208/204824368-9d8accdc-a13f-47bc-995f-483394bc8a29.png)
2.  ![Screenshot_20221130_163642](https://user-images.githubusercontent.com/1178208/204824409-1dbd5b50-cb71-48df-9fa8-4c4d8a7de1c5.png)

**Comment by rdbende** [comment]
No, please don't close this. I wanna fix it. I use a Tkinter-based IDE on my Debian system, and it's very annoying that I can't really see where do I save the file when using a dark ttk theme.

I left some notes for myself in #35 about your first point, although that solution doesn't seem to work.

Unfortunately I have no idea how to flatten the borders of the dropdown menu, but I would be a nice improvement.

**Comment by foozzi** [comment]
> No, please don't close this. I wanna fix it. I use a Tkinter-based IDE on my Debian system, and it's very annoying that I can't really see where do I save the file when using a dark ttk theme.
> 
> 
> 
> I left some notes for myself in #35 about your first point, although that solution doesn't seem to work.
> 
> 
> 
> Unfortunately I have no idea how to flatten the borders of the dropdown menu, but I would be a nice improvement.

Hi @rdbende 
Did you find any solutions ?
I research tcl lang and everything related to [tk_getOpenFile](https://www.tcl.tk/man/tcl/TkCmd/getOpenFile.html#M6), but nothing results
